core/src/test/scala/com/salesforce/op/stages/impl/feature/TextTransmogrifyTest.scala

```diff
@@ -58,14 +60,19 @@ class TextTransmogrifyTest extends FlatSpec with PassengerSparkFixtureTest {
   val (ds, city, country, postal, text, textarea) = TestFeatureBuilder("city", "country", "postal", "text",
     "textarea", data)
 
+  val largeData: Seq[(Text, TextArea)] = largeTextData.zip(largeTextAreaData)
+  val (largeDS, largeText, largeTextarea) = TestFeatureBuilder("largerText", "largerTextarea", largeData)
+
+
   "TextVectorizers" should "vectorize various text feature types" in {
-    val feature = Seq(city, country, postal, text).transmogrify()
+    val feature = Seq(city, country, postal, text, textarea).transmogrify()
     val vectorized = new OpWorkflow().setResultFeatures(feature).transform(ds)
     val vectCollect = vectorized.collect(feature)
 
+    // all text features turned into categoricals since 10 < TextTokenizer.MaxCategoricalCardinality (30)
     for {vector <- vectCollect} {
-      vector.v.size < TransmogrifierDefaults.DefaultNumOfFeatures + (TransmogrifierDefaults.TopK + 2) * 3 shouldBe true
-      vector.v.size >= TransmogrifierDefaults.DefaultNumOfFeatures + 6 shouldBe true
+      vector.v.size < (TransmogrifierDefaults.TopK + 2) * 5  shouldBe true
+      vector.v.size >= 10 shouldBe true
```

Comment:
@sxd929 should there be also a computed value instead of `10`?

Reply:
@sxd929  also please use a better matcher syntax: `vector.v.size should be >= 10`, since it surfaces the error better.

Reply:
fixed! thanks!!